src/compiler/scala/tools/nsc/settings/StandardScalaSettings.scala

```diff
@@ -66,8 +84,14 @@ trait StandardScalaSettings { _: MutableSettings =>
   // Support passe prefixes of -target values:
   //   - `jvm-` (from back when we also had `msil`)
   //   - `1.` (from back when Java 2 was a possibility)
-  // `-target:1.jvm-13` is ridiculous, though.
-  private[this] def normalizeTarget(in: String): String = in.stripPrefix("jvm-").stripPrefix("1.")
+  private def normalizeTarget(in: String): String = {
+    val jvmish = raw"jvm-(\d*)".r
+    in match {
+      case "1.8" | "jvm-1.8" => "8"
```

Comment:
I've just realized that we should also raise deprecation warnings for these first 2 cases as scala 3 accepts only bare numbers `8`, `9`, ...

Reply:
They will find out quickly.